packages/web/src/pages/upload-page/pages/SelectPage.module.css

```diff
@@ -27,3 +27,7 @@
   flex-grow: 0.0001;
   margin: 0;
 }
+
+.audioQuality {
```

Comment:
hmm what's going on here? negative margins are pretty smelly

Reply:
needs to be within the dropzone. tricky to do it without negative margin and without fiddling with the dropzone. dropzone is used in bunch of places which do not care about lossless file quality at all, so thought i'd keep this component out of it and position it properly